tests/forte/data/data_store_test.py

```diff
@@ -128,67 +125,137 @@ def setUp(self) -> None:
         self.reference_type_attributes = {
             "ft.onto.base_ontology.Document": {
                 "attributes": {
-                    "document_class": {"index": 4, "type": (list, (str,))},
-                    "sentiment": {"index": 5, "type": (dict, (str, float))},
+                    "begin": {"index": 2, "type": (None, (int,))},
```

Comment:
Recalled that Suqi mentioned the test case is kinda duplicated in these two test cases, we should centralize them. Have we created tickets for that?

Reply:
Here you go - https://github.com/asyml/forte/issues/894